manual/manual/library/libbigarray.etex

```diff
@@ -2,167 +2,31 @@
 \pdfchapterfold{-1}{The bigarray library}
 %HEVEA\cutname{libbigarray.html}
 
-The "bigarray" library implements large, multi-dimensional, numerical
-arrays.  These arrays are called ``big arrays'' to distinguish them
-from the standard OCaml arrays described in
-\ifouthtml
-\ahref{libref/Array.html}{Module \texttt{Array}}.
-\else
-section~\ref{Array}.
-\fi
-The main differences between ``big arrays''  and standard OCaml arrays
-are as follows:
-\begin{itemize}
-\item Big arrays are not limited in size, unlike OCaml arrays
-("float array" are limited to 2097151 elements on a 32-bit platform,
-other "array" types to 4194303 elements).
-\item Big arrays are multi-dimensional.  Any number of dimensions
-between 1 and 16 is supported.  In contrast, OCaml arrays are
-mono-dimensional and require encoding multi-dimensional arrays as
-arrays of arrays.
-\item Big arrays can only contain integers and floating-point
-numbers, while OCaml arrays can contain arbitrary OCaml data types.
-However, big arrays provide more space-efficient storage of integer
-and floating-point elements, in particular because they support
-``small'' types such as single-precision floats and 8 and 16-bit
-integers, in addition to the standard OCaml types of double-precision
-floats and 32 and 64-bit integers.
-\item The memory layout of big arrays is entirely compatible with that
-of arrays in C and Fortran, allowing large arrays to be passed back
-and forth between OCaml code and C / Fortran code with no data copying
-at all.
-\item Big arrays support interesting high-level operations that normal
-arrays do not provide efficiently, such as extracting sub-arrays and
-``slicing'' a multi-dimensional array along certain dimensions, all
-without any copying.
-\end{itemize}
-%
-Programs that use the "bigarray" library must be linked as follows:
+The "bigarray" library has now been integrated into OCaml's standard
+library.
+
+The "bigarray" functionality may now be found in the standard library
+"Bigarray" module, except for the "map_file" function which is now
+part of the "Unix" module. The documentation has been integrated into
```

Comment:
`"Unix" library` might be more precise and avoid some confusion.  (and ideally it might be nice to add a label to the library description and link to it).

Reply:
Will change it to library.

I don't know how to add such a label (hevea is not my area of expertise), but if you tell me how, I will do it.

Reply:
It now says `"Unix" library`. If you can tell me the rest of the magic for linking etc. I'll add it.

Reply:
First, it would require to add a label to libunix.etex: `label{c:unix}` just after the hevea cutname, then it should be possible to add a link by replacing `"Unix"` with `\ahrefloc{c:unix}{Unix}`.

Reply:
Is there a good way to hyperlink the Bigarray library itself? It clearly has to be different since that file is generated by ocamldoc.

Reply:
hrm. it seems `\ahrefloc` isn't a valid directive.

Reply:
@Octachron did you mean just `\ref`?

Reply:
Sorry this is a html-only macro, `\hyperref[c:unix]{Unix}` should work on both side

Reply:
For linking to a generated API documentation, you can use `\ahref{url}{name}` in a html only section and a reference on the latex side (`(see \ref{Bigarray}` should link to the stdlib Bigarray module).

Reply:
Very weird. `\ahrefloc` was erroring for me before and now it is working. I'm highly confused, but I'm going to quit thinking about it too hard.

Reply:
And now it is failing again. Nothing changed. WTF?

```
! Undefined control sequence.
l.12 part of the \ahrefloc
                          {c:unix}{Unix library}. The documentation has
```

Reply:
Like I said above, `\hyperref[ref]{text}` works.

Reply:
The manual clearly explains this is a real thing, see http://hevea.inria.fr/doc/manual018.html#hevea_default59

And yet, it doesn't work?

Reply:
Okay, I figured it out. Latest version should link the legacy description properly both to the Unix library chapter and to the libref docs for Bigarray.